src/useq/_mda_sequence.py

```diff
@@ -166,6 +188,16 @@ def validate_mda(cls, values: Dict[str, Any]) -> Dict[str, Any]:
                 z_plan=values.get("z_plan"),
                 stage_positions=values.get("stage_positions", ()),
                 channels=values.get("channels", ()),
+                tile_plan=values.get("tile_plan"),
+            )
+
+        if not values.get("stage_positions", ()) and isinstance(
+            values.get("tile_plan"), TileRelative
+        ):
+            raise ValueError(
```

Comment:
hmmm... we don't demand this for z-stack range around do we?  I think you can just assume the position is (0, 0) ... or check what we do in the case of range around without a position

Reply:
for the z we assume it is 0 at the moment.

```python
if self.z_plan.is_relative:
    # TODO: either disallow without position z, or add concept of "current"
    z_pos += getattr(position, Z, None) or 0
```

But I thought it could be good to raise an error for both the tile and the z to avoid any stage movement that could potentially do some damage...even if for z=0 for inverted microscopes usually means that the objective is far for the sample - but what about right microscope? I'm not sure if 0 is far or close to the sample. And maybe for an XY stage it can be the same, where is (0, 0)?

Reply:
> But I thought it could be good to raise an error for both the tile and the z to avoid any stage movement that could potentially do some damage..

you're thinking too much about the runtime behavior :) 

this library is an implementation agnostic *schema*.  It's not a python library for driving acquisition safely.  It knows nothing about how this experiment is run, and shouldn't be opinionated like that.  That kind of error should go in the thing that _runs_ an mda sequence ... or, you could also put that sort of check in the user interface that builds the MDA sequence

Reply:
I'd say use the same "0" pattern as we do for z for now.  I can definitely imagine a downstream library making good use of that.  we shouldn't block it here

Reply:
at the moment we do that:
```python
x_pos = x_pos + dx if x_pos else None
y_pos = y_pos + dy if y_pos else None
```
Do you think it is ok or we need to make `x_pos` and `y_pos` = `0` if there is not a `stage_position`?

Reply:
actually it should be like that:
```python
x_pos = x_pos + dx if x_pos is not None else None
y_pos = y_pos + dy if y_pos is not None else None
```
but we can do like this instead, as you said it might be better:

```python
if tile:
    x_pos, y_pos = (x_pos or 0, y_pos or 0)
    if tile.get("is_relative"):
        # e.g. TileRelative
        dx = tile["x"]
        dy = tile["y"]
        x_pos = x_pos + dx
        y_pos = y_pos + dy
    else:
        # e.g. TileFromCorners
        x_pos = tile.get("x")
        y_pos = tile.get("y")
```